### PR TITLE
Fixed the EMS involved and now it search for EMS too

### PR DESCRIPTION
**Pull request by Annalouu**

### CODE REVIEW

File: ui/app.js [hunk 1]
```diff
@@ -3868,6 +3868,9 @@ $(document).ready(() => {
         $("#reports-officers-involved-tag-title").html(
           "Officers Involved"
         );
+        $("#bolos-officers-involved-tag-tittle").html(
```
Comment (MonkeyWhisper) [inline ui/app.js]: This is duplicated from above

File: ui/app.js [hunk 2]
```diff
@@ -3921,7 +3924,12 @@ $(document).ready(() => {
         $(".incidents-nav-item").hide();
         $(".dmv-nav-item").hide();
         $(".cams-nav-item").hide();
-        $("#reports-officers-involved-tag-title").html("EMS Involved");
+        $("#reports-officers-involved-tag-title").html(
```
Comment (MonkeyWhisper) [inline ui/app.js:3927]: I really dislike that it uses the same handling as reports for officers and bolos...

File: ui/app.js [hunk 3]
```diff
@@ -3921,7 +3924,12 @@ $(document).ready(() => {
         $(".incidents-nav-item").hide();
         $(".dmv-nav-item").hide();
         $(".cams-nav-item").hide();
-        $("#reports-officers-involved-tag-title").html("EMS Involved");
+        $("#reports-officers-involved-tag-title").html(
+          "EMS Involved"
+        );
+        $("#bolos-officers-involved-tag-tittle").html(
```
Comment (MonkeyWhisper) [inline ui/app.js]: There's a typo here, it says tittle

File: ui/dashboard.html [hunk 4]
```diff
@@ -617,7 +617,7 @@
                 </div>
                 <div class="manage-officers-tags-container">
                     <div style="display: flex; align-items: center;">
-                        <div class="tags-title">Officers Involved</div>
+                        <div class="tags-title" id="bolos-officers-involved-tag-tittle">Officers Involved</div>
```
Comment (MonkeyWhisper) [inline ui/dashboard.html]: There's a typo here, it says tittle

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]

File: ui/app.js (hunk 1 above)
Comment (Annalouu) [reply ui/app.js]: not duplicated
one is incident and the other is bolos

**Pull request merged**